docs/CONTRIBUTING.md

```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
     poetry run mypy .
     ```
 
-4.  Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+4. Test!
```

Comment:
I think first we should mention that they need to write tests and where to write them? i.e. add tests to the tests/ directory, which matches the structure of mirascope/ etc.

There is already a Testing section below, maybe we should pull that up here and merge it into this section?

Reply:
that section is too wordy to be bullet point, i will add where to write tests and colocation